packages/react-native-eudiw-pid/src/sd-jwt/types.ts

```diff
@@ -0,0 +1,87 @@
+type CountryCode = string;
+type EvidenceType = string;
+type VerificationAssuranceLevel = string;
+type OIDCFederationEntityStatement = unknown;
+type JWK = {
+  kty: "RSA";
+  use: "sig";
+  n: string;
+  e: string;
+  kid: string;
+};
```

Comment:
Why only RSA? Also kid is optional.
I think for these common types maybe we need to have a common library to import them from!

Reply:
> Why only RSA? Also kid is optional.

That's not just any JWK, it's the key to confirm the possession from the User. Hence, I think `kid` is also unnecessary. I'm gonna both rename the type and implement your suggestions, but I would not care too much for now if you agree.

----

> I think for these common types maybe we need to have a common library to import them from!

Yes I agree, I just wanted a placeholder for now. That's why I didn't pay much attention to detail, to be honest.

Reply:
done in https://github.com/pagopa/io-wallet/pull/11/commits/f13c1374c099447448543304be18d0d0acd9f924

Reply:
... and https://github.com/pagopa/io-wallet/pull/11/commits/58ce9f4216cf9220a1c578d899111b3085ce2f12 😅